components/pd_client/src/client.rs

```diff
@@ -145,18 +149,23 @@ impl RpcClient {
 
     /// Gets the leader of PD.
     pub fn get_leader(&self) -> Member {
-        self.leader_client.get_leader()
+        self.pd_client.get_leader()
     }
 
     /// Re-establishes connection with PD leader in synchronized fashion.
     pub fn reconnect(&self) -> Result<()> {
-        block_on(self.leader_client.reconnect())
+        block_on(self.pd_client.reconnect())
     }
 
     /// Creates a new call option with default request timeout.
     #[inline]
-    fn call_option() -> CallOption {
-        CallOption::default().timeout(Duration::from_secs(REQUEST_TIMEOUT))
+    fn call_option(receiver_addr: &str) -> CallOption {
+        let mut builder = MetadataBuilder::with_capacity(1);
+        builder.add_str("receiver", receiver_addr).unwrap();
```

Comment:
Should be `pd-receiver` to distinguish between pd requests and tikv requests.

Reply:
I think we don't need to distinguish them. We can use the same way to handle the requests of PD/TiKV on the PD side.